docs/en/index.md

```diff
@@ -130,7 +130,7 @@ $articles = $entityManager->getDao(App\Article::getClassName());
 
 $article = new Article();
 $article->title = "The Tigger Movie";
-$articles->save($article);
+$articles->publish($article);
```

Comment:
If you wanna help, change the usage to EntityManager->flush(), I don't wanna advertise the save method anyway :)